triggers/jmods.wc2.pbs

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
+#PBS -N obsproc_%JTYP%_%DESC%_%CC%
+#PBS -o /lfs/h2/emc/stmp/Praveen.Kumar/obsproc_%JTYP%_%PDY%_%CC%_%DESC%.o%J
+#PBS -e /lfs/h2/emc/stmp/Praveen.Kumar/obsproc_%JTYP%_%PDY%_%CC%_%DESC%.o%J
+#PBS -l walltime=1:00:00
+#PBS -A OBSPROC-DEV
+#PBS -q dev 
+#PBS -l select=1:ncpus=2:mem=4gb
+
+## WCOSS1 
+##BSUB -L /bin/sh
+##BSUB -cwd /lfs/h2/stmp/%U
+##BSUB -q dev_transfer ## Change the name 
+##BSUB -M 5000
+##BSUB -R rusage[mem=5000]
+##BSUB -R affinity[core]
```

Comment:
Normally I would advise to strip these out, but maybe leave them in as quick reference for the SPA.